contrib/babelfishpg_tsql/antlr/TSqlParser.g4

```diff
@@ -3844,11 +3844,34 @@ trigger_column_updated
     : UPDATE LR_BRACKET full_column_name RR_BRACKET
     ;
 
-spatial_methods  // we could expand the entire list here, but it is very long
-    : method=id (LR_BRACKET expression_list? RR_BRACKET)?
-    | NULL_P // no bracket
+spatial_methods
+    : ( geospatial_col | geospatial_func_no_arg | geospatial_func_arg ) (LR_BRACKET expression_list? RR_BRACKET)?
     ;
         
+geospatial_col
```

Comment:
should this be `colname=id DOT STX`?

Reply:
Actually it is just the function name we are defining here rest all (i.e. colname, etc) would be present in the expression part before this.